CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+v1.0.0
+-----
+
+* **Breaking Change:** Will not configure or load the `anytv-i18n` module anymore.
```

Comment:
it still uses anytv-mailer, the part where we auto denied an unlink request and we tell them why.